timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosQuorumCheckingCoalescingFunction.java

```diff
@@ -36,13 +38,13 @@
 public class PaxosQuorumCheckingCoalescingFunction<REQUEST, RESPONSE extends PaxosResponse> implements
         CoalescingRequestFunction<REQUEST, PaxosResponses<RESPONSE>> {
 
-    private final List<CoalescingRequestFunction<REQUEST, RESPONSE>> delegates;
-    private final Map<CoalescingRequestFunction<REQUEST, RESPONSE>, ExecutorService> executors;
+    private final List<? extends CoalescingRequestFunction<REQUEST, RESPONSE>> delegates;
+    private final Map<? extends CoalescingRequestFunction<REQUEST, RESPONSE>, ExecutorService> executors;
```

Comment:
not sure, I've kept it as single because I've been lazy, but I believe there are places where we've passed in single executor vs multiple. In essence I think we need to replicate some of the static methods inside `PaxosQuorumChecker`. i.e. one that takes in a map vs one that takes in a single executor.

Reply:
Most likely will add this when I'm doing the wiring.